ql/src/java/org/apache/hadoop/hive/ql/parse/BaseSemanticAnalyzer.java

```diff
@@ -1438,6 +1442,10 @@ public void setUpdateColumnAccessInfo(ColumnAccessInfo updateColumnAccessInfo) {
     this.updateColumnAccessInfo = updateColumnAccessInfo;
   }
 
+  public Set<String> getUserSuppliedFunctions() {
```

Comment:
Maybe add java doc with a note that it contains user supplied functions - and note that it doesn't contain UDFs defined in views, etc.

Reply:
Ack